osbenchmark/workload/params.py

```diff
@@ -1155,6 +1156,16 @@ def _build_vector_search_query_body(self, vector, efficient_filter=None) -> dict
             query.update({
                 "filter": efficient_filter,
             })
+
+        if self.is_nested:
```

Comment:
if more than one level nested is identified, lets raise an exception and add TODO to fix that later